src/content/docs/apm/agents/manage-apm-agents/app-naming/use-multiple-names-app.mdx

```diff
@@ -15,36 +15,40 @@ redirects:
   - /docs/apm/new-relic-apm/installation-configuration/use-multiple-names-app
 ---
 
-import apmInsightsToggleonandOff from 'images/apm_screenshot-full_insights-toggle-on-and-off.png'
+When working with your APM data in New Relic, you may want to view aggregated, "rolled up" data for an application that spans multiple clusters, environments, or data centers, but also be able to view each of the app instance's data separately. You can do this by setting multiple app names for your APM-monitored apps.
 
-When working with your data in New Relic, you may want to view aggregated data for an application across clusters, environments, or data centers, while at the same time be able to view each of your application instance's data individually.
+## Important notes [#important-notes]
 
-<Callout variant="important">
-  If you use multiple names for an app, the last name assigned is the least specific name New Relic uses to roll up the data. For best results to view both instance-level and aggregated data, set the order of your app's rollup names from **most specific to least specific** in your configuration file.
-</Callout>
+Here are some important caveats to be aware of when using multiple app names.
 
-## Roll up app data [#rollup]
+### Alternative options [#alt-options]
 
-Normally, when two instances report with the same app name, agent language, and [license key](/docs/apis/intro-apis/new-relic-api-keys/#ingest-license-key), New Relic aggregates their data into a single New Relic-monitored app. You can also view data for the individual hosts and instances from the app's New Relic [APM **Overview** page](/docs/apm/applications-menu/monitoring/apm-overview-page#infra-server).
+Because adding multiple app names results in [duplicate data](#duplicate), you may want to consider these other options. 
```

Comment:
@zuluecho9 It seems worth saying why duplicate data is a problem here in addition to linking to that later section.